WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1444,6 +1451,7 @@ private boolean userCanPublishPosts() {
         }
     }
 
+    @SuppressLint("StaticFieldLeak")
```

Comment:
It doesn't look like a false positive and having a non-static inner class which can outlive it's parent can lead to memory leaks. Should we really suppress it? It's part of the lint-baseline.xml, so it shouldn't break the build on Travis and we might want to fix it in the future. WDYT?

Reply:
I agree. I just didn't want to do that in this PR since it's not related. I'll remove the suppressions so we don't forget about it.